### DISCUSSION TITLE
IC-7300 via usb, no transmit audio

### DISCUSSION THREAD

**Discussion by rakar** [N0BOY/FT8CN#57] [Q&A]
My settings decode fine and on transmit the rig goes into transmit mode, but there is no outbound audio. 
Any help would be greatly appreciated. Make this my goto app.

**Comment by bg7yoz** [comment]
hi, @rakar 

请确认一下事情：
1.解码的声音是不是从usb端口的接收到的？7300使用usb接口应该是使用虚拟声卡的，如果您的手机是从usb采集到音频，那么从频谱图可以看到手机是不会采集环境音的。如果没有正确从usb端口接收音频，那么也就不会从usb端口发射音频了。

2.看看发射时，ptt是否已经启动？在ft8cn中请正确选择电台型号，错误的型号可能不会正确启动ptt。ft8cn支持7300，控制模式请选择cat。

希望以上建议能对你有帮助。

bg7yoz
